granulate_utils/linux/cgroups/cgroup.py

```diff
@@ -3,15 +3,20 @@
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
 
-from typing import List, Mapping, Optional, Tuple
+from __future__ import annotations
+
+import os
+from abc import abstractmethod
+from pathlib import Path
+from typing import List, Mapping, Optional
 
 import psutil
 
 from granulate_utils.linux import ns
 from granulate_utils.linux.mountinfo import iter_mountinfo
 from granulate_utils.linux.process import read_proc_file
 
-SUBSYSTEMS = {
+CONTROLLERS = {
```

Comment:
The list is different between V2 and V1, how does it work here with you having a single list?

Reply:
I replied to one of your comments below, I think this is irrelevant and should be removed.